HerPortal.BusinessLogic/Models/LocalAuthority.cs

```diff
@@ -1,5 +1,10 @@
 ﻿namespace HerPortal.BusinessLogic.Models;
 
+/// <summary>
+///     This model is not a full model for Local Authority Data.
+///     It should be considered as a reference to the full data for a Local Authority.
+///     The full data (including an LAs relationship to a Consortium) can be found in the HUG2 Public Website codebase
```

Comment:
We do have this data in this codebase too (in `LocalAuthorityData`) - worth referencing that too